modules/database/src/main/java/org/eclipse/xpanse/modules/database/serviceconfiguration/ServiceConfigurationEntity.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.database.serviceconfiguration;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.hypersistence.utils.hibernate.type.json.JsonType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.OffsetDateTime;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Data;
+import org.eclipse.xpanse.modules.database.common.ObjectJsonConverter;
+import org.eclipse.xpanse.modules.database.service.DeployServiceEntity;
+import org.hibernate.annotations.Type;
+import org.springframework.format.annotation.DateTimeFormat;
+
+/**
+ * ServiceConfigurationEntity for persistence.
+ */
+@Table(name = "SERVICE_CONFIGURATION")
+@Entity
+@Data
+public class ServiceConfigurationEntity {
+
+    @Id
+    private UUID id;
```

Comment:
Do we really need an additional primary key here? Since there is only one-to-one mapping to the parent table, we can make the service_id column as the primary key which is anyways unique.